portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormRules.path

```diff
@@ -85,6 +85,11 @@
     <td></td>
 </tr>
 <!--RULE_BUILDER_AUTOFILL-->
+<tr>
+	<td>RULE_BUILDER_AUTOFILL_FIELD_BY_INDEX</td>
```

Comment:
@ewertonluna, is it really necessary on this pull? Can you please clarify?

Reply:
Hi @natocesarrego! It really wasn't necessary, so I removed it.